PROPOSALS.md

```diff
@@ -78,11 +78,29 @@ request feedback on feasibility before investing time and effort in the full pro
     * or just share patch files, in the same format as
       [`head.hackage`](https://gitlab.haskell.org/ghc/head.hackage) does.
 5. Implement the proposal.
-    * Note that you should follow [GHC's "Contributing a Patch" guidelines](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch) when creating an MR (e.g. [appropriate commit messages](https://gitlab.haskell.org/ghc/ghc/-/wikis/Contributing-a-Patch#22-commit-messages)),
+    * Register at https://gitlab.haskell.org/ghc/ghc.
+    * If your account is not approved soon (this is a manual process to combat spam),
+      ask for approval in your CLC discussion thread, someone will take care of it.
+    * Fork GHC repository and implement your proposal in a non-`master` branch.
+      Do not work in `master`, it will cause issues with Marge Bot later.
```

Comment:
Can we find a different way of phrasing this? The branch that the implementer works in in their own local repo is of no consequence. I doubt that what the implementer pushes to their own GitLab fork is of consequence either.

```suggestion
    * Fork GHC repository and implement your proposal in a non-`master` branch.
      (It's probably best if you create a new branch rather than working `master`, see below.)
```

Reply:
> I doubt that what the implementer pushes to their own GitLab fork is of consequence either.

It is of consequence, Marge Bot refuses to cooperate if you create an MR from `master` branch of your fork. E. g., https://gitlab.haskell.org/ghc/ghc/-/merge_requests/11714#note_538135.

Reply:
Right, what you create the MR from is of consequence, whether or not you happen to have a `master` branch pushed to your fork is not.